server/src/main/java/org/apache/iotdb/db/engine/compaction/selector/impl/SizeTieredCompactionSelector.java

```diff
@@ -142,12 +147,25 @@ private boolean selectLevelTask(
     // if next time partition exists
     // submit a merge task even it does not meet the requirement for file num or file size
     if (hasNextTimePartition && selectedFileList.size() > 1) {
-      taskPriorityQueue.add(new Pair<>(new ArrayList<>(selectedFileList), selectedFileSize));
+      addOneTaskToQueue(taskPriorityQueue, selectedFileList, selectedFileSize);
```

Comment:
The return value is not used.

Reply:
It is unnecessary to use return value of `addOneTaskToQueue` here, because no matter what it is, `selectLevelTask` method will return false here.